README.md

```diff
@@ -6,8 +6,8 @@ Easy-to-use, modular web framework for V.
 ```v
 module main
 
-import vex.server // or nedpals.vex.server
-import vex.ctx // or nedpals.vex.ctx
+import nedpals.vex.server
+import nedpals.vex.ctx
```

Comment:
The commented alternative import name was intentional for those who use vpm or installed the package globally (for vpkg). Maybe add a little note or exchange the two of them to use the `nedpals.vex.server`/`ctx` as the de facto way to import modules.